### PR TITLE
focal-point.js unit-tests

### PR DESCRIPTION
**Pull request by Flight**
- focal-point.js unit-tests
- gulp file updated
- package.json updated
- jquery simulate added

### CODE REVIEW

**Comment by dhorelik** [comment]
lgtm

File: gulpfile.js [hunk 1]
```diff
@@ -67,24 +66,28 @@ gulp.task('jshint', function () {
...
         singleRun: true
     }, done).start();
 });
+gulp.task('tests:watch', function () {
+    gulp.watch(PROJECT_PATTERNS.lint, ['tests:unit']);
+});
 
+// #############################################################################
+// TASKS
+gulp.task('js', gulpsync.sync(['jshint', 'jscs', 'tests:unit']));
```
Comment (vxsx) [inline gulpfile.js:83]: why do you need to run them synchronously? for the output on ci?

**Comment by vxsx** [comment]
:+1:

File: gulpfile.js (hunk 1 above)
Comment (Flight) [inline gulpfile.js:83]: Yes, it slightly improves output on ci and during local tests and doesn't affect the speed a lot.

**Pull request merged**